package.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>google_benchmark_vendor</name>
+  <version>0.9.0</version>
```

Comment:
Why start with this version number? I would expect 0.0.0 until the first release is being made.

Reply:
No reason, really. I'm happy to reset to 0.0.0.

Reply:
3d0ed57